### DISCUSSION TITLE
انشاء قواعد الترجمة

### DISCUSSION THREAD

**Discussion by aissat** [softvenue/i18n#79] [Ideas]
السلام عليكم 
لدي بعض اقتراح حسب وجهة نظري من مهم العمل عليه قبل بداية العمل على ترجمة الكلمات هو :
**العمل على انشاء و وضع اسس وقواعد الترجمة** 
لان العمل على هذا النقطة  راح يظبط الترجمة و كيفية التعامل مع الكلمات 
مثلا لو اخذنا مثال في #76 صراحة لايوجد اي مصطلح عربي اصيل لكلمة  backend  لذلك سيكون امامنا الكثير من اقترحات ف بدون قواعد تظبط ترجمة على اي اساس نختار الترجمة الادق والاقرب للمعنى 
في اخير اضن ان العمل على وضع قواعد و اسس قياسية عمل كبير و راح  يفيد المحتوى العربي اكثر بحيث يكون مرجع موحد لكل مشاريع الترجمة 
@softvenue/all

**Comment by celyes** [comment]
يبدو اقتراحاً جيداُ
سأبحث عن مصادر لاساسيات وضع المصطلحات من الناحية اللغوية و أوافيكم بها هنا حتى نكون على قدر المهمة.
شكرا لك

**Comment by celyes** [comment]
تفضلوا بقراءة المقال: 
[قواعد وضع المصطلحات](https://darfikr.com/article/%D8%B9%D9%84%D9%85-%D8%A7%D9%84%D9%85%D8%B5%D8%B7%D9%84%D8%AD-%D9%88%D8%B7%D8%B1%D8%A7%D8%A6%D9%82-%D9%88%D8%B6%D8%B9-%D8%A7%D9%84%D9%85%D8%B5%D8%B7%D9%84%D8%AD%D8%A7%D8%AA-%D9%81%D9%8A-%D8%A7%D9%84%D8%B9%D8%B1%D8%A8%D9%8A%D8%A9)
